### ISSUE TITLE
Suggest widget: details are not read out by NVDA

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52450]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Open any file in the editor and type any word.

**Actual:**
NVDA does not read the details attached to each list item in the suggestion treeview.

**Expected:**
NVDA should also read the details or description related to each item along with the list item.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Editor NVDA does not read details.pptx](https://github.com/Microsoft/vscode/files/2118956/A11y.VSCode.Editor.NVDA.does.not.read.details.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I feel like there should be some user triggered action in oreder for the details to be read out. Because if we start reading all the details for each suggestion that will be too much text.

Due to that and other rendering reasons I prefer our previous suggest solution where we showed the details only when user explictly clicked on them. Just my 2 cents
fyi @joaomoreno

**Comment by ramya-rao-a** [comment]
When we say "details" here are we referring to 

![image](https://user-images.githubusercontent.com/16890566/42478664-6a53d6dc-838a-11e8-948e-528db5a5ac37.png)

or 

![image](https://user-images.githubusercontent.com/16890566/42478678-7a2ade16-838a-11e8-9e64-1eec9b5def44.png)

If its the latter, then that view is attained by user interaction. They need to press Ctrl+Space once the suggest widget is visible. In this case all that text should be read.

**Comment by isidorn** [comment]
I believe the first case is the issue that accessibility team outlined here.
Not sure if it should be fixed though, leaving up to you to decide

**Comment by ramya-rao-a** [comment]
In the below case, it is intentional that we dont read the details out. There is no sensible separator we can add between the actual item and this detail. Therefore, it will be way too much text to go through before moving to the next item.

![image](https://user-images.githubusercontent.com/16890566/42478664-6a53d6dc-838a-11e8-948e-528db5a5ac37.png)

In the above case, the screenreader reads the item and follows up with "has details". If user presses Ctrl+Space at this point, the docs expand to the side and all the details are read.

From this point onwards, the docs are always expanded and will be read as well
